### ISSUE TITLE
wazuh kibana elasticsearch

### ISSUE THREAD

**Issue by cx13283081880** [wazuh/wazuh-kibana-app#5891]
Could you  tell me how they work together？
How Kibana and ElasticSearch work inside Wazuh？

**Comment by AlexRuiz7** [comment]
Hello @cx13283081880 

We currently use OpenSearch and OpenSearch Dashboards as a base, instead of Elasticsearch and Kibana, but they are almost the same as OpenSearch is a fork of Elastic.

In summary, Wazuh uses Elasticsearch as a database and backend, and Kibana as a User Interface to represent the data.

To better understand how these components integrate into Wazuh's environment, I encourage you to read the [Getting started](https://documentation.wazuh.com/current/getting-started/index.html) section of our documentation. Note that we don't mention Elastic or OpenSearch, but Wazuh indexer and Wazuh dashboard (Elastic and Kibana so to say), as we understood that by doing so the users got confused. You can still see those references in older versions of the documentation ([< 4.2](https://documentation.wazuh.com/4.2/getting-started/components/index.html)).


I hope I explained it clearly, let me know if you have more questions.

Regards,
Alex

**Comment by cx13283081880** [comment]
Thanks for your reply.
We now want to add some functionality to the wazuh web interface.
Can you tell me exactly where the relevant code is?
Looking forward to your reply.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "wazuh/wazuh-kibana-app"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月15日(星期五) 下午5:50
***@***.***&gt;;
抄送:&nbsp;"陈曦 ***@***.******@***.***&gt;;
主题:&nbsp;Re: [wazuh/wazuh-kibana-app] wazuh kibana elasticsearch (Issue #5891)





 
Hello @cx13283081880
 
We currently use OpenSearch and OpenSearch Dashboards as a base, instead of Elasticsearch and Kibana, but they are almost the same as OpenSearch is a fork of Elastic.
 
In summary, Wazuh uses Elasticsearch as a database and backend, and Kibana as a User Interface to represent the data.
 
To better understand how these components integrate into Wazuh's environment, I encourage you to read the Getting started section of our documentation. Note that we don't mention Elastic or OpenSearch, but Wazuh indexer and Wazuh dashboard (Elastic and Kibana so to say), as we understood that by doing so the users got confused. You can still see those references in older versions of the documentation (< 4.2).
 
I hope I explained it clearly, let me know if you have more questions.
 
Regards,
 Alex

**Comment by AlexRuiz7** [comment]
Depends on what you want to add / change.

Our main plugin is located in this repo

**Comment by AlexRuiz7** [comment]
Hello @cx13283081880 

I'm closing the issue due to inactivity.

Feel free to continue the thread, and we'll re-open the issue, or create a new one.

Regards,
Alex
